src/main/io/gps.c

```diff
@@ -1578,9 +2070,51 @@ static bool UBLOX_parse_gps(void)
     uint32_t i;
 
     *gpsPacketLogChar = LOG_IGNORED;
+#define CLSMSG(cls, msg) (((cls) << 8) | (msg))
+    switch (CLSMSG(_class, _msg_id)) {
+    case CLSMSG(CLASS_MON, MSG_MON_VER): {
+#ifdef USE_GPS_DEBUG
+        char *token = strtok(_buffer.ver.swVersion, " ");
+        while (token != NULL) {
+            float f = fastA2F(token);
+            if (f > 0.0f) {
+                int major = strtol(token, NULL, 10);
+                int minor = strtol(&token[major > 9 ? 3 : 2], NULL, 10);
+                gpsData.monVer.swVersion.firmwareVersion.major = major;
+                gpsData.monVer.swVersion.firmwareVersion.minor = minor;
+                break;
+            }
+            token = strtok(NULL, " ");
+        }
+
 
-    switch (_msg_id) {
-    case MSG_POSLLH:
+        // get software protocol version details
+        #define TAG "PROTVER="
+        for(unsigned j = 0; j < 10; j++) {
+            const char* p = (&_buffer.ver.extension[j * 30]);
```

Comment:
```suggestion
            char* p = (&_buffer.ver.extension[j * 30]);
```

Reply:
Marking as resolved since this code has been removed and can be discussed in a later PR.